src/applications/vaos/new-appointment/components/calendar/CalendarWidget.jsx

```diff
@@ -281,7 +287,12 @@ export default function CalendarWidget({
 
 CalendarWidget.propTypes = {
   additionalOptions: PropTypes.object,
-  availableDates: PropTypes.array, // ['YYYY-MM-DD']
+  availableSlots: PropTypes.arrayOf(
```

Comment:
You could maybe simplify the interface to the calendar and accept a list of date strings, but seemed like it could be useful in the future to have both the start and end date, and it was less work to leave this structure in place.